### ISSUE TITLE
:zoom-on-pinch="false" not work

### ISSUE THREAD

**Issue by LuRM** [bcakmakoglu/vue-flow#242]
I set the property zoom-on-pinch to false, but is it still possible to zoom in and out when pinching (touch or trackpad).
code example:
```
<vue-flow
    ref="vueFlow"
    @dragover="onDragOver"
    class="vue-flow-container"
    v-model="elements"
    :zoom-on-pinch="false"
    :zoom-on-double-click="false"
  >
    <template #edge-custom="props">
      <custom-edge v-bind="props" @handleAdd="handleAdd" @onDrop="btnOnDrop" />
    </template>
  </vue-flow>
```

**Comment by bcakmakoglu** [comment]
Thanks for reporting!

That is a bug, yea. Unfortunately the watcher assumes it shouldn't watch falsy values.
Will be fixed with the next patch.

**Comment by LuRM** [comment]
:zoom-on-double-click="false" doesn't work either

**Comment by bcakmakoglu** [comment]
@LuRM fixed with `0.4.31`
